spaceship/docs/DeveloperPortal.md

```diff
@@ -1,35 +1,41 @@
 # Developer Portal API
 
-- [Usage](#usage)
-  * [Login](#login)
-  * [Apps](#apps)
-    + [App Services](#app-services)
-  * [App Groups](#app-groups)
-  * [iCloud Containers](#icloud-containers)
-  * [Apple Pay Merchants](#apple-pay-merchants)
-  * [Passbook](#passbook)
-  * [Certificates](#certificates)
-    + [Code Signing Certificates](#code-signing-certificates)
-    + [Push Certificates](#push-certificates)
-    + [Create a Certificate](#create-a-certificate)
-  * [Provisioning Profiles](#provisioning-profiles)
-    + [Receiving profiles](#receiving-profiles)
-    + [Create a Provisioning Profile](#create-a-provisioning-profile)
-    + [Repair all broken provisioning profiles](#repair-all-broken-provisioning-profiles)
-  * [Devices](#devices)
-  * [Enterprise](#enterprise)
-  * [Multiple Spaceships](#multiple-spaceships)
-  * [More cool things you can do](#more-cool-things-you-can-do)
-  * [Example Data](#example-data)
-- [License](#license)
+- [Developer Portal API](#developer-portal-api)
```

Comment:
Large amount of code shift here. Best I can tell its from adding "Developer Portal API" as a top level item, which is basically this table of contents itself. We could reduce churn here by removing that to return existing table of contents with just adding the new "Authentication Keys" section

Reply:
Fixed